<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deposit</class>
 <widget class="QWidget" name="deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>339</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>339</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>339</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Deposit</string>
  </property>
  <widget class="QLineEdit" name="depositor_name_input">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>20</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="depositor_name_text">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>21</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Depositor's name</string>
   </property>
  </widget>
  <widget class="QLabel" name="depositor_cpf_text">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>45</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Depositor's CPF</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="depositor_cpf_input">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>44</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>11</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="value_text">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>94</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Value</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_input">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>93</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="favoured_account_text">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>70</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>favoured account</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="favoured_account_input">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>69</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="error_message">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>242</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="deposit_button">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>282</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Deposit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>282</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_text">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>181</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_output">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>180</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="destination_information_text">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>150</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Destination information</string>
   </property>
  </widget>
  <widget class="QLabel" name="cpf_output">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>205</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cpf_text">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>205</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CPF</string>
   </property>
  </widget>
  <widget class="QLabel" name="account_text">
   <property name="geometry">
    <rect>
     <x>207</x>
     <y>207</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Account</string>
   </property>
  </widget>
  <widget class="QLabel" name="account_output">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>206</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="check_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>52</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: orange;
 border-style: solid;
color: black;
border-radius: 30px</string>
   </property>
   <property name="text">
    <string>Check</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>depositor_name_input</tabstop>
  <tabstop>depositor_cpf_input</tabstop>
  <tabstop>favoured_account_input</tabstop>
  <tabstop>value_input</tabstop>
  <tabstop>check_button</tabstop>
  <tabstop>deposit_button</tabstop>
  <tabstop>cancel_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
